src/frontend/index.ts

```diff
@@ -160,9 +154,19 @@ export async function createFlowrWindow(flowrStore: Store): Promise<FlowrWindow>
 
       evt.sender.send('receiveConfig', config)
     },
-    getMacAddress: async (evt: any) => {
-      const usedMacAddress = await getMacAddress()
-      evt.sender.send('receiveMacAddress', usedMacAddress)
+    getActiveMacAddress: async (evt: any) => {
+      const activeMacAddress = await getActiveMacAddress()
+      evt.sender.send('receiveActiveMacAddress', activeMacAddress)
```

Comment:
To guarantee backward compatibility, we should also send `evt.sender.send('receiveMacAddress', activeMacAddress)`

Reply:
(it would also return the active mac address)